tensorflow/python/ops/check_ops.py

```diff
@@ -1504,6 +1505,206 @@ def assert_type(tensor, tf_type, message=None, name=None):
     return control_flow_ops.no_op('statically_determined_correct_type')
 
 
+@tf_export('debugging.assert_shapes', 'assert_shapes')
```

Comment:
Can you run the api compatibility test in tensorflow/tools/api/tests:api_compatibility_test and follow the instructions to regenerate the golden files?